src/mixer/basetrackplayer.cpp

```diff
@@ -107,6 +107,15 @@ BaseTrackPlayerImpl::BaseTrackPlayerImpl(QObject* pParent,
             this,
             &BaseTrackPlayerImpl::slotCloneFromDeck);
 
+    // Sampler cloning
+    m_pCloneFromSampler = std::make_unique<ControlObject>(
+            ConfigKey(getGroup(), "CloneFromSampler"),
```

Comment:
From the [Wiki](https://github.com/mixxxdj/mixxx/wiki/mixxxcontrols):

> Newly added controls should use the snake_case naming convention.

Reply:
I know but in this case I chose to keep it consistent with existing `CloneFromDeck`

Reply:
I could also change both the existing control to clone_from_... and insert an alias for CloneFrom..
That way we have all controls in basetrackplayer in snake_case and custom mappings are not affected.

Reply:
yeah, I'd add an alias:
```c++

    // Deck cloning
    ConfigKey* cloneTargetGroup = ConfigKey(getGroup());
    m_pCloneFromDeck = std::make_unique<ControlObject>(
            cloneTargetGroup, "clone_from_deck"),
            false);
    connect(m_pCloneFromDeck.get(),
            &ControlObject::valueChanged,
            this,
            &BaseTrackPlayerImpl::slotCloneFromDeck);
    ControlDoublePrivate::insertAlias(ConfigKey(cloneTargetGroup, "CloneFromDeck"),
                                      ConfigKey(cloneTargetGroup, "clone_from_deck"));

    // Sampler cloning
    m_pCloneFromSampler = std::make_unique<ControlObject>(
            ConfigKey(getGroup(), "clone_from_sampler"),
            false);
```
Okay?

Reply:
No. let's stick with the camel case version to not clutter the Cos even more.

Reply:
Alright. So I'm done here.

Reply:
eeehm, forgot to push..
Done.